src/unix/bsd/openbsdlike/openbsd.rs

```diff
@@ -392,6 +440,12 @@ extern {
                        serv: *mut ::c_char,
                        servlen: ::size_t,
                        flags: ::c_int) -> ::c_int;
+    pub fn kevent(kq: ::c_int,
+                  changelist: *const ::kevent,
+                  nchanges: ::size_t,
+                  eventlist: *mut ::kevent,
+                  nevents: ::size_t,
+                  timeout: *const ::timespec) -> ::c_int;
```

Comment:
`nchanges` and `nevents` should be `c_int`.
But I am unsure about the error message of OpenBSD test: it seems related to return type ?